7-yoda/1-example.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class Order
+{
+    public function updateOrder()
+    {
+        // Typical conditional structure
+        if ( $postType == 'post' ) {
+            // do stuff
+        }
+
+        // This is a typo, since it should be '==' and not '='
+        if ( $postType = 'post' ) {
+            // do stuff
+        }
+
+        // Yoda conditions
+        if ( 'post' == $postType ) {
```

Comment:
you can additionally mention when you explain to use `===` for php and js...

Reply:
didn't get you